packages/experiments/src/components/PersonaCoin/PersonaCoinInitials/PersonaCoinInitials.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { Icon } from 'office-ui-fabric-react';
+import { Text } from '../../Text/Text';
```

Comment:
This should be `../../../Text` for consistency

Reply:
Fixed